src/dotnet/commands/dotnet-build/BuildCommand.cs

```diff
@@ -37,6 +37,8 @@ public static BuildCommand FromArgs(string[] args, string msbuildPath = null)
 
             var appliedBuildOptions = result["dotnet"]["build"];
 
+            msbuildArgs.Add($"/clp:Summary");
```

Comment:
This is so that this can be overwritten?

Reply:
Correct. As it was, dotnet build /clp:NoSummary was broken.